jquery.liteuploader.js

```diff
@@ -154,16 +176,13 @@ LiteUploader.prototype = {
     _buildXhrObject: function () {
         var xhr = new XMLHttpRequest();
         xhr.upload.addEventListener('progress', this._onXHRProgress.bind(this), false);
+        this.xhrs.push(xhr);
         return xhr;
     },
 
-    _getXHRObject: function () {
-        return this.xhr;
-    },
-
     _performUpload: function (formData) {
         $.ajax({
-            xhr: this._getXHRObject.bind(this),
+            xhr: this._buildXhrObject.bind(this),
```

Comment:
Every upload creates it's own `XMLHttpRequest` instance on the fly. The build function adds the instance to the `xhrs` container.